core/consensus/common/src/lib.rs

```diff
@@ -120,6 +120,9 @@ impl<T: SyncOracle> SyncOracle for Arc<T> {
 
 /// A list of all well known keys in the cache.
 pub mod well_known_cache_keys {
+	/// The type representing cache keys.
+	pub type Id = [u8; 4];
```

Comment:
@bkchr Perhaps tuple struct should be used instead? I have used `pub type ConsensusEngineId = [u8; 4];` as an example.

Reply:
That is enough, we don't need to use a tuple struct here 👍